app/posts/apply-for-teacher-training/2021-07-23-end-of-cycle-2021.md

```diff
@@ -0,0 +1,226 @@
+{{ appFigure({
+  image: {
+    file: "normal-email-apply-again.png"
+  },
+  caption: "Candidates get the usual 'apply again' email inviting them to apply again in the same recruitment cycle."
+}) }}
+
+{% from "figure/macro.njk" import appFigure with context %}
+{{ appFigure({
+  image: {
+    file: "application-unsuccessful-still-time-to-apply.png"
+  },
+  caption: "If they log in before the deadline to apply again, they'll see the usual 'apply again' message but with the addition of a deadline banner. They can apply again in the normal way, up until the deadline to apply again."
+}) }}
+
+### If it's too late to apply in the same recruitment cycle, they need to 'carry over' their application
+
+{% from "figure/macro.njk" import appFigure with context %}
+{{ appFigure({
+  image: {
+    file: "application-unsuccessful-no-longer-possible-apply-again.png"
+  },
+  caption: "As soon as the deadline to apply again has passed, the message in the 'apply again' inset text changes - letting the candidate know they can apply for the next cycle. Clicking 'apply again' carries over their application."
+}) }}
+
+{% from "figure/macro.njk" import appFigure with context %}
+{{ appFigure({
+  image: {
+    file: "apply-again-next-cycle.png"
+  },
+  caption: "If the application does not lead to a place when the deadline to apply again has already passed, the usual 'apply again' email changes to reflect this."
+}) }}  
+
+[If the candidate carries their application over before Apply reopens, they cannot submit](/apply-for-teacher-training/end-of-cycle-2021/#if-they-carry-over-before-apply-reopens%2C-they-cannot-submit) - as above.
+
+[If the candidate carries their application over before Find reopens, they cannot find courses](/apply-for-teacher-training/end-of-cycle-2021/#if-they-carry-over-before-find-reopens%2C-they-cannot-find-courses) - as above.
+
+## Candidates who create an account after the deadline to apply for the first time
+
+[If the candidate creates an account before Apply reopens, they cannot submit](/apply-for-teacher-training/end-of-cycle-2021/#if-they-carry-over-before-apply-reopens%2C-they-cannot-submit) - as above.
+
+[If the candidate creates an account before Find reopens, they cannot find courses](/apply-for-teacher-training/end-of-cycle-2021/#if-they-carry-over-before-find-reopens%2C-they-cannot-find-courses) - as above.
+
+## Timeframe to carry over an application
+
+We have not set an end point for carrying over an application.
+
+This means that the [carry over message when a candidate's application does not lead to a place](/apply-for-teacher-training/end-of-cycle-2021/#if-it%E2%80%99s-too-late-to-apply-in-the-same-recruitment-cycle%2C-they-need-to-%E2%80%98carry-over%E2%80%99-their-application) and the [unsubmitted carry over message](/apply-for-teacher-training/end-of-cycle-2021/#they-need-to-%E2%80%98carry-over%E2%80%99-their-application-to-continue-working-on-it) continue to show indefinitely until the candidate actions it.
```

Comment:
> continue to show indefinitely until the candidate actions it.

A small and possibly irrelevant detail, but not sure it’s indefinitely. Is there not a point when we purge unsubmitted applications (after one complete cycle without activity on an application)?

Reply:
@paulrobertlloyd so I say this later: 'We’re assessing whether we should set an end point.' - cos I thought we were figuring this out atm?